lib/dns.js

```diff
@@ -31,7 +72,7 @@ Client.prototype.query = function query(type, host, cb) {
     ra: true
   });
 
-  if (host[host.length - 1] !== '.')
+  if (host[ host.length - 1 ] !== '.')
```

Comment:
yea, i think it's these:

  "requireSpacesInsideArrayBrackets": "all",
  "requireSpacesInsideObjectBrackets": "all",

i'm all for getting rid of these two rules :)